src/main/scala/com/databricks/spark/sql/perf/mllib/feature/Tokenizer.scala

```diff
@@ -0,0 +1,28 @@
+package com.databricks.spark.sql.perf.mllib.feature
+
+import org.apache.spark.ml
+import org.apache.spark.ml.linalg.Vector
+import org.apache.spark.ml.PipelineStage
+import org.apache.spark.sql._
+
+import com.databricks.spark.sql.perf.mllib.OptionImplicits._
+import com.databricks.spark.sql.perf.mllib.data.DataGenerator
+import com.databricks.spark.sql.perf.mllib.{BenchmarkAlgorithm, MLBenchContext, TestFromTraining}
+
+/** Object for testing Tokenizer performance */
+object Tokenizer extends BenchmarkAlgorithm with TestFromTraining with UnaryTransformer {
+
+  override def trainingDataSet(ctx: MLBenchContext): DataFrame = {
+    import ctx.params._
+    import ctx.sqlContext.implicits._
+
+    DataGenerator.generateRandString(ctx.sqlContext,
+      numExamples, ctx.seed(), numPartitions, 100, 200, "abcdefg ")
+      .select($"label".as(inputCol))
```

Comment:
This data generation code can be improved.
Maybe I can reuse some data generation code for `LDA` ?
@smurching Do you have some written code for this?